### ISSUE TITLE
Question: Dimension of proprioception representation

### ISSUE THREAD

**Issue by Junxix** [Max-Fu/icrt#5]
Hi, thank you for sharing your research and code!

I noticed a potential discrepancy between the code and the paper. In the code([convert_droid_icrt.py](https://github.com/Max-Fu/icrt/blob/main/tools/convert_droid_icrt.py)) I see:
`obs_gp.create_dataset(f'cartesian_position',(max_epi_len,6))`
However, in your paper (page 10), you mention a "10-dimensional proprioception representation". I have two questions:

1. For fine-tuning, what dimensionality should we use?
2. During pre-training, was the gripper range normalized to [0,1]?

Thanks for your help in clarifying these points!

**Comment by Max-Fu** [comment]
Thank you for your interest in this work! 

> proprio representation

Droid has cartesian xyz and euler angle (6dof) as the proprio / action representations. In the code we will convert them to xyz + 6d angle representation (first two rows of rotation matrix). 

Please refer to paper appendix E for more detail.

> Gripper 

We directly regress to the gripper width and did not apply normalization. 

Lmk if you have more questions!

**Comment by Junxix** [comment]
Thanks for your reply!

**Comment by Junxix** [comment]
By the way, have you tried using abs_action as the prediction target?

**Comment by Max-Fu** [comment]
I remember predicting absolute actions was not as good relative in terms of task performance, but we don't have the exact delta in success rate.
